### PR TITLE
show current logged in user. ref item:1258

### PR DESCRIPTION
**Pull request by jofomah**

### CODE REVIEW

**Comment by tlvince** [comment]
This is wrapping onto a new line in Safari:

screen shot 2015-02-27 at 07 27 15

File: src/components/navbar/navbar.html [hunk 1]
```diff
@@ -23,4 +23,12 @@
       </ul>
     </div>
   </div>
+
+  <div ng-if="navbarCtrl.auth.isLoggedIn" class="navbar-right nav-user-pad">
+    <span class="white-text">
+      <i class="fa fa-user"></i>
+      <span ng-bind="navbarCtrl.auth.currentUser.name"></span>
```
Comment (tlvince) [inline src/components/navbar/navbar.html]: You could add one-time binding.

File: src/components/navbar/navbar.html [hunk 2]
```diff
@@ -23,4 +23,12 @@
       </ul>
     </div>
   </div>
+
+  <div ng-if="navbarCtrl.auth.isLoggedIn" class="navbar-right nav-user-pad">
+    <span class="white-text">
```
Comment (tlvince) [inline src/components/navbar/navbar.html]: You can add the whole block within a new `<li>` in the `navbar-right` list above?

Comment (jofomah) [inline src/components/navbar/navbar.html]: am chaning it to use drop down and also one time binding. will send in the PR when am done.

Comment (tlvince) [inline src/components/navbar/navbar.html]: :+1:

**Comment by jofomah** [comment]
@tlvince see update.

**Comment by tlvince** [comment]
Nice one. Can you drop the unused styles?

**Comment by jofomah** [comment]
which ones?

**Comment by tlvince** [comment]
`nav-user-pad` and `white-text`

**Comment by tlvince** [comment]
+1

**Pull request merged**